dev/ansible/05_trino_create_tables.yml

```diff
@@ -76,7 +76,7 @@
     - test
 
 - name: Create CSV table
-  shell: docker run --network host -v "/vagrant/example/resources/query/csv_create_table.sql:/csv_create_table.sql" prestosql/presto:341 presto --http-proxy 127.0.0.1:8888 --server 127.0.0.1:8888 --catalog hive --schema default --user presto --file /csv_create_table.sql
+  shell: docker run --network host -v "/vagrant/example/resources/query/csv_create_table.sql:/csv_create_table.sql" trinodb/trino:354 trino --http-proxy 127.0.0.1:8888 --server 127.0.0.1:8888 --catalog hive --schema default --user trino --file /csv_create_table.sql
```

Comment:
We should consider extracting `trinodb/trino:354` to a ansible variable. Used alot of places.